nginx/default.conf

```diff
@@ -3,7 +3,8 @@
 # Redirect http://  to https://
 server {
     server_name localhost;
-    access_log off;
+    # access_log off;
+    access_log /var/log/nginx/access_min.log custom;
```

Comment:
Looks good, but I don't think we use this file anymore. We should update in [digi-serve/ab_service_web](https://github.com/digi-serve/ab_service_web)